src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Presentation.Api.OData/WorkplacesController/Translations.g.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Generated
+
+#nullable enable
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OData.Deltas;
+using Microsoft.AspNetCore.OData.Query;
+using Microsoft.AspNetCore.OData.Results;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using Microsoft.EntityFrameworkCore;
+using MediatR;
+using Nox.Application;
+using Nox.Extensions;
+
+using System;
+using System.Net.Http.Headers;
+using ClientApi.Application;
+using ClientApi.Application.Dto;
+using ClientApi.Application.Queries;
+using ClientApi.Application.Commands;
+using ClientApi.Domain;
+using ClientApi.Infrastructure.Persistence;
+
+namespace ClientApi.Presentation.Api.OData;
+         
+
+
+
+public abstract partial class WorkplacesControllerBase
+{
+    
+    [HttpPut("api/Workplaces/{key}/WorkplaceLocalized/{cultureCode}")]
+    public virtual async Task<ActionResult<WorkplaceLocalizedDto>> PutWorkplaceLocalized( [FromRoute] System.UInt32 key, [FromRoute] System.String cultureCode, [FromBody] WorkplaceLocalizedUpsertDto workplaceLocalizedUpsertDto)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+        
+        var updatedProperties = new Dictionary<string, dynamic>();
+        var etag = Request.GetDecodedEtagHeader();
```

Comment:
need from the "Workplace" entity => GetWorkplaceById(...).Select(e=> e.Etag)

Reply:
Desired changes has been applied